conf/nginx/templates/nginx.conf.web.clamav.signature.provider.template

```diff
@@ -0,0 +1,22 @@
+# Carbonio ClamAV Signature Provider proxy default configuration
+#
+server
+{
+    listen 10000;
+    listen [::]:10000;
```

Comment:
Also this is exposing signatures on port 10000 on all interfaces. I think we should just expose them on 127.78.0.X, to expose them only internally.

Reply:
It can cause conflicts with other services also, due to binding on all interfaces. carbonio-files for example already runs on 10000

Reply:
We start service on 127.78.0.21:10000 now as discussed. :+1: